README.md

```diff
@@ -2,3 +2,39 @@
 Community contributions for STAT GR 5702 Fall 2019
 
 This is an amazing class.
```

Comment:
You can delete the "amazing class" line. 

Add some explanation about bookdown and a link to an example bookdown web site.